app-editors/qhexedit2/qhexedit2-0.8.4.ebuild

```diff
@@ -11,7 +11,7 @@ inherit distutils-r1 qmake-utils
 
 DESCRIPTION="Hex editor library, Qt application written in C++ with Python bindings"
 HOMEPAGE="https://github.com/lancos/qhexedit2/"
-SRC_URI="https://github.com/lancos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/lancos/${PN}/archive/ponyprog.tar.gz -> ${P}.tar.gz"
```

Comment:
Are we only masquerading as qhexedit2 now?

Reply:
qhexedit2 has branch ponyprog. in first ebuild version i was select incorrect branch, ponyprog 3.0.0 can't build with it.

Reply:
In fact this becomse -9999.ebuild, beacuse you are pointing it to the HEAD of ponyprog branch.
Also please note that master and ponyprog branches are even in terms of commits.

Reply:
And by -9999 I mean that it will also break next time somebody will commit to that branch with "hash mismatch".

Reply:
this still has not been adressed